### DISCUSSION TITLE
能从首页的“添加游戏”进入网站后台是很正常的吗？

### DISCUSSION THREAD

**Discussion by bluesiboy** [Gamuxorg/bbs#398] [游戏或网站运行反馈]
RT

**Comment by hyh19962008** [comment]
对的，欢迎大家投稿

Best

________________________________
From: 蓝多 ***@***.***>
Sent: Monday, October 31, 2022 5:00:25 PM
To: Gamuxorg/bbs ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [Gamuxorg/bbs] 能从首页的“添加游戏”进入网站后台是很正常的吗？ (Discussion #398)


RT

―
